src/server/search-index/images.search-index.ts

```diff
@@ -0,0 +1,182 @@
+import { client } from '~/server/meilisearch/client';
+import { getOrCreateIndex, onSearchIndexDocumentsCleanup } from '~/server/meilisearch/util';
+import { EnqueuedTask } from 'meilisearch';
+import {
+  createSearchIndexUpdateProcessor,
+  SearchIndexRunContext,
+} from '~/server/search-index/base.search-index';
+import { ImageIngestionStatus, MetricTimeframe } from '@prisma/client';
+import { imageSelect } from '~/server/selectors/image.selector';
+
+const READ_BATCH_SIZE = 1000;
+const MEILISEARCH_DOCUMENT_BATCH_SIZE = 100;
```

Comment:
I have seen these a couple times already, maybe we can drop them in `constants.ts` file